tests/unit/s2n_extended_master_secret_test.c

```diff
@@ -18,12 +18,15 @@
 
 #include "utils/s2n_bitmap.h"
 
+#define IS_TLS12_CONNECTION(client, server) \
+    ((client->actual_protocol_version == S2N_TLS12) && (server->actual_protocol_version == S2N_TLS12))
```

Comment:
Since Wesley pointed out that a macro like this probably shouldn't be used with EXPECT_FALSE, it seems a little dangerous to keep around even if it's currently only used with EXPECT_TRUE. I'd just make the two assertions. If you really don't want to write them both every time, you could also do:
```suggestion
#define EXPECT_TLS12_CONNECTIONS(client, server) \
    EXPECT_EQUAL(client->actual_protocol_version, S2N_TLS12); \
    EXPECT_EQUAL(server->actual_protocol_version, S2N_TLS12)
```